tools/src/versioning/android/transforms/expoviewTransforms.ts

```diff
@@ -91,6 +91,11 @@ export function expoviewTransforms(abiVersion: string): FileTransforms {
         find: /versioned\/host\/exp\/exponent\/modules\/api\/reanimated/g,
         replaceWith: `${abiVersion}/host/exp/exponent/modules/api/reanimated`,
       },
+      {
+        paths: './**/reanimated/NativeProxy.java',
+        find: /\b(com\.swmansion\.)/g,
+        replaceWith: `${abiVersion}.$1`,
+      },
```

Comment:
versioning the gesturehandler imports in reanimated's NativeProxy.java. we need this because gesturehandler imports are not in `host.exp.exponent.modules.api` package after new style vendoring anymore.